autoloads/Globals.gd

```diff
@@ -69,3 +70,13 @@ func reset():
 		ResourceType.PARTS: 0,
 		ResourceType.RESEARCH: 0
 	}
+	
+func set_difficulty(difficulty: DifficultyLevel):
+	match difficulty:
+		DifficultyLevel.EASY:
+			BASE_DOME_INFESTATION_RATE = .025
+		DifficultyLevel.NORMAL:
+			BASE_DOME_INFESTATION_RATE = .05
+		DifficultyLevel.HARD:
+			BASE_DOME_INFESTATION_RATE = .075
+	BASE_INFESTATION_FIGHT_RATE = -BASE_DOME_INFESTATION_RATE
```

Comment:
doesn't this mean that easy will actually be harder that normal, since FIGHT_RATE is lower than all the other speeds?

Reply:
Hmm, good point. Leaving fight rate unchanged from normal will mean infestation can be fought more easily, rather than everything just being slowed down